src/omero_zarr/cli.py

```diff
@@ -87,6 +87,23 @@ def _configure(self, parser):
             type=ProxyStringType("Image"),
             help="The Image from which to export Masks.",
         )
+        masks.add_argument(
+            "--split-masks",
+            action="store_true",
+            help=(
+                "Store each ROI in a separate group, required if masks "
+                "overlap"
+            ),
+        )
+        masks.add_argument(
+            "--mask-bits",
+            default=str(max(MASK_DTYPE_SIZE.keys())),
```

Comment:
You're picking `max` because the safety doesn't cost much because it's compressed when on disk?